Lib/fontTools/pens/basePen.py

```diff
@@ -36,34 +36,42 @@
 sequence of length 2 will do.
 """
 
-from typing import Tuple
+from typing import Any, Dict, List, Optional, Tuple, Union
+from typing_extensions import TypeAlias
 
 from fontTools.misc.loggingTools import LogMixin
 
-__all__ =  ["AbstractPen", "NullPen", "BasePen", "PenError",
-			"decomposeSuperBezierSegment", "decomposeQuadraticSegment"]
+__all__ = [
+	"AbstractPen", "NullPen", "BasePen", "PenError", "decomposeSuperBezierSegment",
+	"decomposeQuadraticSegment", "PenPoint",
+]
+
+
+PenGlyphSet: TypeAlias = Optional[Dict[str, Any]]
+PenPoint: TypeAlias = Tuple[float, float]
```

Comment:
I don't agree that a "pen point" is a an x, y tuple. It is a sequence with length two, containing two numbers.